LICENSE

```diff
@@ -44,7 +44,7 @@
       of this License, Derivative Works shall not include works that remain
       separable from, or merely link (or bind by name) to the interfaces of,
       the Work and Derivative Works thereof.
- 
+
```

Comment:
I fear now you added the space at another place :) I think you are right though, this shouldn’t be any problem. I don’t mind merging as is, but if you could revert the two whitespace changes, that’d be nice